apps/dashboard/test/unit/rclone_util_test.rb

```diff
@@ -0,0 +1,16 @@
+require "rclone_helper"
```

Comment:
Just for styling it would be best to use `require 'rclone_helper' for consistency.